.eslintrc

```diff
@@ -1,9 +1,8 @@
 {
     "env": {
-        "browser": true,
-        "commonjs": true,
```

Comment:
I think those are included in the nodejs environment.

http://eslint.org/docs/user-guide/configuring#specifying-environments says that commonjs - CommonJS global variables and CommonJS scoping (use this for browser-only code that uses Browserify/WebPack).``